pricelist_cache/models/product_pricelist.py

```diff
@@ -3,12 +3,50 @@
 
 from datetime import date
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class Pricelist(models.Model):
     _inherit = "product.pricelist"
 
+    parent_pricelist_ids = fields.Many2many(
+        "product.pricelist",
+        relation="product_pricelist_cache__parent_pricelist_ids_rel",
+        column1="pricelist_id",
+        column2="parent_pricelist_id",
+        compute="_compute_parent_pricelist_ids",
+        store=True,
```

Comment:
it's a m2m, it's already stored

Reply:
are you sure ?
I get this:
```
❯ odoo -d testdb -i pricelist_cache --stop-after-init
[...]
❯ psql testdb
psql (15.1 (Ubuntu 15.1-1.pgdg18.04+1), server 12.13 (Ubuntu 12.13-1.pgdg18.04+1))
Type "help" for help.

testdb=> select * from product_pricelist_cache__parent_pricelist_ids_rel;
ERROR:  relation "product_pricelist_cache__parent_pricelist_ids_rel" does not exist
LINE 1: select * from product_pricelist_cache__parent_pricelist_ids_...
```

Reply:
bah, sorry, I didn't consider is computed :rofl: